kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -147,7 +147,8 @@
   );
   private static final ConfigDef.Importance KEY_SOURCE_IMPORTANCE = ConfigDef.Importance.MEDIUM;
   private static final String KEY_SOURCE_DOC =
-          "Determines whether the keyfile config is the path to the credentials json, or the json itself";
+          "Determines whether the keyfile config is the path to the credentials json, the json itself " +
+                  "or a service account (only in GCP)";
```

Comment:
I think we should be clear here that this new option isn't just for service accounts, it can be used for application default credentials anywhere. It could be useful to note that a common use case for application default credentials is service accounts but if someone wants to learn more about how this works then the term "Application default credentials" is a bit easier to search for (AFAICT) than just "GCP service account".

Reply:
Agree, changed